finish/src/main/java/io/openliberty/guides/inventory/model/SystemData.java

```diff
@@ -14,15 +14,20 @@
 
 import java.util.Properties;
 import org.eclipse.microprofile.openapi.annotations.media.Schema;
-
```

Comment:
Keep this spacing between imports and beginning of class annotation

Reply:
Updated.